packages/auth/amplify_auth_cognito_dart/lib/src/flows/device/confirm_device_worker.dart

```diff
@@ -67,6 +66,25 @@ abstract class ConfirmDeviceWorker
   /// {@macro amplify_auth_cognito.confirm_device_worker}
   factory ConfirmDeviceWorker.create() = ConfirmDeviceWorkerImpl;
 
+  @override
+  String get jsEntrypoint {
+    if (zIsFlutter && zReleaseMode) {
```

Comment:
Hmm not sure- could be a melos vs. non-melos issue if you were running `dart/flutter pub get` at some point. Moot point now since this is generated and uses aws_common